src/components/DataCard/DataCard.tsx

```diff
@@ -0,0 +1,48 @@
+import { type } from "os";
+import React from "react";
+import {
+  DataCardContainer,
+  DataCardHeader,
+  BottomLineStyle,
+} from "./DataCard-style";
+import chartImg from "../../assets/Icons/chart.svg";
+import { colors } from "../../Utils/Constances";
+
+type IDataCard = {
```

Comment:
Which other props it should get?

Reply:
The graphs and content, but so far there is only the image titled "No Data"